frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -995,20 +995,26 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                                       'Cities',
                                       WebStatsBreakdown.City
                                   ),
-                                  createTableTab(
-                                      TileId.GEOGRAPHY,
-                                      GeographyTab.TIMEZONES,
-                                      'Timezones',
-                                      'Timezones',
-                                      WebStatsBreakdown.Timezone
-                                  ),
                                   createTableTab(
                                       TileId.GEOGRAPHY,
                                       GeographyTab.LANGUAGES,
                                       'Languages',
                                       'Languages',
                                       WebStatsBreakdown.Language
                                   ),
+
+                                  // :BUG:, see #26475, hidden behind FF while we fix it
+                                  ...[
+                                      featureFlags[FEATURE_FLAGS.WEB_ANALYTICS_TIMEZONE_TAB]
+                                          ? createTableTab(
+                                                TileId.GEOGRAPHY,
+                                                GeographyTab.TIMEZONES,
+                                                'Timezones',
+                                                'Timezones',
+                                                WebStatsBreakdown.Timezone
+                                            )
+                                          : null,
+                                  ].filter((x) => x !== null),
```

Comment:
Yes, it's complex, TS wouldn't let me otherwise

Reply:
It's temporary, so I'm fine with how bad this looks

Reply:
We have `isNotNil` as a type guard, which can work well for this kind of thing. You might be able to add `.filter(isNotNil)` to the outer array, and let this be null without needing to create an inner array

Reply:
That didn't work on its own, but I changed the code slightly, it looks nicer now

https://github.com/PostHog/posthog/pull/26478/commits/a4c395616257ebc6e6ce76a8e7dd83a273377516